wg-iot-edge/README.md

```diff
@@ -17,7 +17,7 @@ A Working Group dedicated to discussing, designing and documenting using Kuberne
 ## Meetings
 * APAC WG Meeting: [Wednesdays at 5:00 UTC](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit) (every four weeks). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=5:00&tz=UTC).
   * [Meeting notes and Agenda](https://docs.google.com/document/d/1Yuwy9IO4X6XKq2wLW0pVZn5yHQxlyK7wdYBZBXRWiKI/edit?usp=sharing).
-* Regular WG Meeting: [Wednesdays at 09:00 (Pacific Time) PT](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit) (every four weeks). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=09:00 (Pacific Time)&tz=PT).
+* Regular WG Meeting: [Wednesdays at 08:00 (Pacific Time) PT](https://www.google.com/calendar/event?eid=Y25ibTA0Mm1kb2tycXZpNTNiazA2bDR0czhfMjAxODEwMTJUMTUwMDAwWiBjZ250MzY0dmQ4czg2aHIycGhhcGZqYzZ1a0Bn&ctz=America/Los_Angeles) (every four weeks). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=08:00 (Pacific Time)&tz=PT).
```

Comment:
This links to the event back in 2018 when the meeting was on Friday. I think it can confuse people and it's better to add a link to the whole calendar, where they can see easily the next event

https://calendar.google.com/calendar/embed?src=sensatic.com_05l7huufn6lrq9np0idvt6gk6g%40group.calendar.google.com&&ctz=America/Los_Angeles

Also the change from 9AM to 8AM is wrong

Reply:
You're correct - it was the only meeting in the Kubernetes calendar, and thus the only one I found, and I hadn't noticed it was an old recurring meeting (and not a still-recurring one).

Is there a specific reason wg-iot-edge doesn't use the Kubernetes calendar?

Reply:
(I'm referring to [this one](https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America/Los_Angeles))

Reply:
@configurator The regular meeting time is on the calendar (there's the one tomorrow). For some reason APAC meeting is not showing up. I just shared the calendar with community@kubernetes.io, so let's see if that will help. If you have any other ideas on how to improve this, please let me know.

Reply:
@configurator my bad, APAC one is there as well, just on Tuesday evening as it is shown in Pacific timezone.